maintainers/maintainer-list.nix

```diff
@@ -951,6 +951,11 @@
     matrix = "@alexshpilkin:matrix.org";
     name = "Alexander Shpilkin";
   };
+  AlexSKaye = {
+    github = "AlexSKaye";
```

Comment:
An e-mail would be nice.
I suggest you to create a dedicated ProtonMail for GitHub- and other forges- related things.